lib/generators/clearance/specs/templates/features/clearance/visitor_resets_password_spec.rb.tt

```diff
@@ -3,6 +3,15 @@ require "support/features/clearance_helpers"
 
 RSpec.feature "Visitor resets password" do
   before { ActionMailer::Base.deliveries.clear }
+<% if defined?(ActiveJob) -%>
+
+  around do |example|
+    original_adapter = ActiveJob::Base.queue_adapter
+    ActiveJob::Base.queue_adapter = :inline
+    example.run
+    ActiveJob::Base.queue_adapter = original_adapter
```

Comment:
I can do that. Is there a scenario where an exception not automatically handled by the example itself wouldnt immediately cause the test suite to crash?